CHANGELOG

```diff
@@ -1,3 +1,7 @@
+# 1.2.0
+- feat: get
+
+
```

Comment:
please keep single empty line in changelog to keep same standard with other releases